packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.jsx

```diff
@@ -85,6 +85,10 @@ export const IdvDocSubmitOnSignup = ({ citizen_data, has_previous, onPrevious, o
             setInputDisable(false);
         }
 
+        if (document_number === document_type.example_format) {
+            errors.document_number = localize('Please enter a valid ID number different from example.');
```

Comment:
Where is this content coming from?

Reply:
Currently it is just a temporary text. I asked for clarification, ask for code review in order to not lose time. I mentioned it in announce_reviews_fe